### ISSUE TITLE
[Community pipeline] Echo-Memory overlay for Wan 2.1 1.3B

### ISSUE THREAD

**Issue by WayneJin0918** [huggingface/diffusers#14482]
### Model/Pipeline/Scheduler description

Request to add a **GitHub community pipeline** that overlays the released Echo-Memory `context_k1` DiT fine-tune onto official Diffusers Wan 2.1 1.3B.

This is not a new official Wan checkpoint and not the full Echo-Memory multi-chunk / camera-action / SSM research stack. It remaps original DiffSynth/Wan transformer keys onto `WanPipeline.transformer` (825 / 825 official keys) so users can load:

```python
from diffusers import DiffusionPipeline

pipe = DiffusionPipeline.from_pretrained(
    "Wan-AI/Wan2.1-T2V-1.3B-Diffusers",
    custom_pipeline="pipeline_echo_memory",
    torch_dtype=torch.bfloat16,
)
pipe.load_echo_memory_weights()
```

PR: https://github.com/huggingface/diffusers/pull/14471

### Open source status

- [x] The model implementation is available.
- [x] The model weights are available (Only relevant if addition is not a scheduler).

### Provide useful links for the implementation

- Paper: https://arxiv.org/abs/2606.09803
- Code: https://github.com/Echo-Team-Joy-Future-Academy-JD/Echo-Memory
- Original keys: https://huggingface.co/Echo-Team/Echo-Memory (`context_k1/epoch-0.safetensors`)
- Converted transformer + Hub custom pipeline: https://huggingface.co/Wayne-King/echo-memory-diffusers
- Authors: @WayneJin0918 / Echo Team

**Comment by WayneJin0918** [comment]
I'm one of the Echo-Memory authors (@WayneJin0918). Thanks for the pointer — a Hub repo is already live, so people can use this without waiting on a GitHub merge:

https://huggingface.co/Wayne-King/echo-memory-diffusers

```python
import torch
from diffusers import DiffusionPipeline

pipe = DiffusionPipeline.from_pretrained(
    "Wan-AI/Wan2.1-T2V-1.3B-Diffusers",
    custom_pipeline="Wayne-King/echo-memory-diffusers",
    torch_dtype=torch.bfloat16,
    trust_remote_code=True,
)
pipe.load_echo_memory_weights()  # remaps Echo-Team/Echo-Memory `context_k1`
# or: pipe.load_converted_echo_memory_weights()
```

What this is:
- official Wan 2.1 1.3B Diffusers pipeline + one released DiT row (`context_k1`)
- 825 / 825 transformer keys remapped from original DiffSynth / Wan names
- not the full multi-chunk / camera-action / SSM research stack

@asomoza happy to take whichever path you prefer:

1. Keep it as a Hub `custom_pipeline` (already usable)
2. Port to Modular Diffusers custom blocks if you want it in the collection
3. Keep or close the `examples/community` PR https://github.com/huggingface/diffusers/pull/14471

Friction so far: Echo-Memory checkpoints use original Wan / DiffSynth keys, so a remap (or a converted file) is required before `WanPipeline.transformer.load_state_dict`. `scripts/convert_wan_to_diffusers.py` is not importable from a community file, so we duplicated the T2V rename dict.
